templates/cluster_resources.j2

```diff
@@ -119,7 +138,10 @@ primitive rsc_sap_{{ sid }}_ERS{{ ers_instance }} SAPInstance \
   meta priority=1000
 
 group grp_{{ sid }}_ERS{{ ers_instance }} \
-  rsc_ip_{{ sid }}_ERS{{ ers_instance }} rsc_fs_{{ sid }}_ERS{{ ers_instance }} rsc_sap_{{ sid }}_ERS{{ ers_instance }} {% if cloud_provider == "microsoft-azure" %} nc_{{ sid }}_ERS {% endif %}
+  rsc_ip_{{ sid }}_ERS{{ ers_instance }} rsc_fs_{{ sid }}_ERS{{ ers_instance }} rsc_sap_{{ sid }}_ERS{{ ers_instance }}
```

Comment:
is it possible to add a comment on this ( i mean all the template file looks human unreadable and ulgy).. :man_shrugging:

Reply:
I'm not sure what are you asking here. What should I comment? I don't even know what's the comment syntax on a crmsh config file. :D

Reply:
ok if ther is no comment possible is ok